docs/framework/data/adonet/ef/language-reference/compiled-queries-linq-to-entities.md

```diff
@@ -66,4 +66,4 @@ When you have an application that executes structurally similar queries many tim
 
 - [ADO.NET Entity Framework](../index.md)
 - [LINQ to Entities](linq-to-entities.md)
-- [Entity Framework Merge Options and Compiled Queries](https://go.microsoft.com/fwlink/?LinkId=199591)
+- [Entity Framework Merge Options and Compiled Queries](https://docs.microsoft.com/archive/blogs/)
```

Comment:
```suggestion
- [Entity Framework Merge Options and Compiled Queries](https://docs.microsoft.com/en-us/archive/blogs/dsimmons/ef-merge-options-and-compiled-queries)
```

Reply:
Updated your suggestion to have correct link text and no locale

Reply:
Thanks.. too much copy paste going on in my head hehe